tests/Unit/Scraper/Listeners/UpdateDatasetTest.php

```diff
@@ -16,15 +15,13 @@ class UpdateDatasetTest extends TestCase
 {
     use DatabaseMigrations;
 
-    private UpdateDataset $updateDataset;
+    private UpdateDatasets $updateDataset;
```

Comment:
Why renaming this class? UpdateDatasets does not exist.

Reply:
This was done because we were not able to access the public functions of factory class and were only getting the protected functions of the class which was initialized in the constructor of factory class inside vendor library. 

There was some conflict with the property name and some other in built library variable. When we changed the name then we were able to access the create function of factory class and then things started working.

Reply:
You renamed `UpdateDataset` to `UpdateDatasets`, you are using a class that does not exist now, how can this work? We only have  `Joskfg\LaravelIntelligentScraper\Scraper\Listeners\UpdateDataset`.

Sorry, but I don’t get which private methods about a factory class you are talking about. `UpdateDataset` is a listener.